### PR TITLE
CXSPA-716: Review Deleted API

### PR DESCRIPTION
**Pull request by plabadie**
closes https://jira.tools.sap/browse/CXSPA-716

### CODE REVIEW

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts [hunk 1]
```diff
@@ -1436,43 +1436,43 @@ export const GENERATED_REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
...
     comment:
-      'Class UserReplenishmentOrderConnector has been removed and is no longer part of the public API. ',
+      'Class UserReplenishmentOrderConnector has been removed and is no longer part of the public API. Use ReplenishmentOrderConnector in @spartacus/order/core.',
   },
   {
     node: 'UserReplenishmentOrderService',
     importPath: '@spartacus/core',
     comment:
-      'Class UserReplenishmentOrderService has been removed and is no longer part of the public API. ',
+      'Class UserReplenishmentOrderService has been removed and is no longer part of the public API. Use ReplenishmentOrderFacade in @spartacus/order/root instead.',
```
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: `ReplenishmentOrderHistoryFacade` (there is `History` in the name)

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts [hunk 2]
```diff
@@ -1436,43 +1436,43 @@ export const GENERATED_REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
...
     comment:
-      'Class UserReplenishmentOrderAdapter has been removed and is no longer part of the public API. ',
+      'Class UserReplenishmentOrderAdapter has been removed and is no longer part of the public API. Use ReplenishmentOrderAdapter in @spartacus/order/core.',
   },
   {
     node: 'UserReplenishmentOrderConnector',
     importPath: '@spartacus/core',
     comment:
-      'Class UserReplenishmentOrderConnector has been removed and is no longer part of the public API. ',
+      'Class UserReplenishmentOrderConnector has been removed and is no longer part of the public API. Use ReplenishmentOrderConnector in @spartacus/order/core.',
```
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: `ReplenishmentOrderHistoryConnector`

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts [hunk 3]
```diff
@@ -1436,43 +1436,43 @@ export const GENERATED_REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
...
     comment:
-      'Class UserOrderService has been removed and is no longer part of the public API. ',
+      'Class UserOrderService has been removed and is no longer part of the public API. Use OrderFacade in @spartacus/order/root instead.',
   },
   {
     node: 'UserReplenishmentOrderAdapter',
     importPath: '@spartacus/core',
     comment:
-      'Class UserReplenishmentOrderAdapter has been removed and is no longer part of the public API. ',
+      'Class UserReplenishmentOrderAdapter has been removed and is no longer part of the public API. Use ReplenishmentOrderAdapter in @spartacus/order/core.',
```
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: `ReplenishmentOrderHistoryAdapter`

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts [hunk 4]
```diff
@@ -1436,43 +1436,43 @@ export const GENERATED_REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
...
     comment:
-      'Class UserOrderConnector has been removed and is no longer part of the public API. ',
+      'Class UserOrderConnector has been removed and is no longer part of the public API. Use OrderConnector in @spartacus/order/core.',
   },
   {
     node: 'UserOrderService',
     importPath: '@spartacus/core',
     comment:
-      'Class UserOrderService has been removed and is no longer part of the public API. ',
+      'Class UserOrderService has been removed and is no longer part of the public API. Use OrderFacade in @spartacus/order/root instead.',
```
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: `OrderHistoryFacade`

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts [hunk 5]
```diff
@@ -1436,43 +1436,43 @@ export const GENERATED_REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
...
     comment:
-      'Class UserOrderAdapter has been removed and is no longer part of the public API. ',
+      'Class UserOrderAdapter has been removed and is no longer part of the public API. Use OrderAdapter in @spartacus/order/core.',
   },
   {
     node: 'UserOrderConnector',
     importPath: '@spartacus/core',
     comment:
-      'Class UserOrderConnector has been removed and is no longer part of the public API. ',
+      'Class UserOrderConnector has been removed and is no longer part of the public API. Use OrderConnector in @spartacus/order/core.',
```
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: `OrderHistoryConnector`

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts [hunk 6]
```diff
@@ -1436,43 +1436,43 @@ export const GENERATED_REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
     node: 'UserOrderAdapter',
     importPath: '@spartacus/core',
     comment:
-      'Class UserOrderAdapter has been removed and is no longer part of the public API. ',
+      'Class UserOrderAdapter has been removed and is no longer part of the public API. Use OrderAdapter in @spartacus/order/core.',
```
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: `OrderHistoryAdapter`

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts [hunk 7]
```diff
@@ -1310,121 +1310,121 @@ export const GENERATED_REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
...
     comment:
-      'Variable USER_RETURN_REQUEST_DETAILS has been removed and is no longer part of the public API. ',
+      "Variable USER_RETURN_REQUEST_DETAILS has been removed and is no longer part of the public API. It is replaced by RETURN_REQUEST_DETAILS from '@spartacus/order/core', but the constant value is not the same.",
   },
   {
     node: 'USER_RETURN_REQUESTS',
     importPath: '@spartacus/core',
     comment:
-      'Variable USER_RETURN_REQUESTS has been removed and is no longer part of the public API. ',
+      "Variable USER_RETURN_REQUESTS has been removed and is no longer part of the public API. It is replaced by RETURN_REQUESTS from '@spartacus/order/core', but the constant value is not the same.",
```
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts:1415]: Screen Shot 2022-07-20 at 4 05 36 PM

`USER_RETURN_REQUESTS` is from `order-state.ts`? It is not there.

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts [hunk 8]
```diff
@@ -1310,121 +1310,121 @@ export const GENERATED_REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
...
     comment:
-      'Class SaveCartConnector has been removed and is no longer part of the public API. ',
+      "Class SaveCartConnector has been removed and is no longer part of the public API. Use CartConnector.save instead from '@spartacus/cart/base/core'",
   },
   {
     node: 'USER_ORDER_DETAILS',
     importPath: '@spartacus/core',
     comment:
-      'Variable USER_ORDER_DETAILS has been removed and is no longer part of the public API. ',
+      "Variable USER_ORDER_DETAILS has been removed and is no longer part of the public API. It is replaced by ORDER_DETAILS from '@spartacus/order/core', but the constant value is not the same.",
```
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts:1385]: check my comment in 1415.

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts [hunk 9]
```diff
@@ -1310,121 +1310,121 @@ export const GENERATED_REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
...
     comment:
-      'Variable MultiCartSelectors.getWishListId has been removed and is no longer part of the public API. ',
+      "Variable MultiCartSelectors.getWishListId has been removed and is no longer part of the public API. Use 'getCartIdByTypeFactory' instead",
   },
   {
     node: 'OccUserOrderAdapter',
     importPath: '@spartacus/core',
     comment:
-      'Class OccUserOrderAdapter has been removed and is no longer part of the public API. ',
+      'Class OccUserOrderAdapter has been removed and is no longer part of the public API. Use OccOrderAdapter in @spartacus/order/occ instead',
```
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts:1355]: `OccReplenishmentOrderHistoryAdapter`

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts [hunk 10]
```diff
@@ -1310,121 +1310,121 @@ export const GENERATED_REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
...
   },
   {
     node: 'OccUserReplenishmentOrderAdapter',
     importPath: '@spartacus/core',
     comment:
-      'Class OccUserReplenishmentOrderAdapter has been removed and is no longer part of the public API. ',
+      'Class OccUserReplenishmentOrderAdapter has been removed and is no longer part of the public API. Use OccReplenishmentOrderAdapter in @spartacus/order/occ instead',
```
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: `OccReplenishmentOrderHistoryAdapter`

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts (hunk 9 above)
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: `OccOrderHistoryAdapter`

File: projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/generated-methods-and-properties.migration.ts [hunk 11]
```diff
@@ -1209,13 +1202,6 @@ export const GENERATED_METHODS_AND_PROPERTIES_MIGRATION: MethodPropertyDeprecati
       comment:
         "The 'getParams' method's signature changed to: 'getParams(): Observable<{\n        [_: string]: any;\n    } | undefined>'",
     },
-    {
-      class: 'DeleteUserAddressEvent',
-      importPath: '@spartacus/core',
-      deprecatedNode: 'userId',
-      comment:
-        "// TODO:Spartacus - Property 'userId' was removed from Class 'DeleteUserAddressEvent'. ",
```
Comment (WeizhengSap) [inline projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/generated-methods-and-properties.migration.ts:1217]: It is not removed. The `userId` is moved to parent event `UserAddressEvent`. So, the children events still have this property.

File: docs/migration/5_0-generated-typescript-changes-doc.md [hunk 12]
```diff
@@ -8947,7 +8927,7 @@ moved to @spartacus/order/occ
 
 
 Class OccUserOrderAdapter has been removed and is no longer part of the public API.
-
+Use OccOrderAdapter in @spartacus/order/occ instead
```
Comment (WeizhengSap) [inline docs/migration/5_0-generated-typescript-changes-doc.md:8931]: `OccOrderHistoryAdapter`

File: docs/migration/5_0-generated-typescript-changes-doc.md [hunk 13]
```diff
@@ -8956,7 +8936,7 @@ Class OccUserOrderAdapter has been removed and is no longer part of the public A
 
 
 Class OccUserReplenishmentOrderAdapter has been removed and is no longer part of the public API.
-
+Use OccReplenishmentOrderAdapter in @spartacus/order/occ instead
```
Comment (WeizhengSap) [inline docs/migration/5_0-generated-typescript-changes-doc.md]: `OccReplenishmentOrderHistoryAdapter`

File: docs/migration/5_0-generated-typescript-changes-doc.md [hunk 14]
```diff
@@ -12554,7 +12524,7 @@ Class UserOccTransitionalModule has been removed and is no longer part of the pu
 
 
 Class UserOrderAdapter has been removed and is no longer part of the public API.
-
+Use OrderAdapter in @spartacus/order/core.
```
Comment (WeizhengSap) [inline docs/migration/5_0-generated-typescript-changes-doc.md:12528]: `OrderHistoryAdapter`

File: docs/migration/5_0-generated-typescript-changes-doc.md [hunk 15]
```diff
@@ -12563,7 +12533,7 @@ Class UserOrderAdapter has been removed and is no longer part of the public API.
 
 
 Class UserOrderConnector has been removed and is no longer part of the public API.
-
+Use OrderConnector in @spartacus/order/core.
```
Comment (WeizhengSap) [inline docs/migration/5_0-generated-typescript-changes-doc.md]: `OrderHistoryConnector`

File: docs/migration/5_0-generated-typescript-changes-doc.md [hunk 16]
```diff
@@ -12572,7 +12542,7 @@ Class UserOrderConnector has been removed and is no longer part of the public AP
 
 
 Class UserOrderService has been removed and is no longer part of the public API.
-
+Use OrderFacade in @spartacus/order/root instead.
```
Comment (WeizhengSap) [inline docs/migration/5_0-generated-typescript-changes-doc.md]: `OrderHistoryFacade`

File: docs/migration/5_0-generated-typescript-changes-doc.md [hunk 17]
```diff
@@ -12631,7 +12601,7 @@ store: Store<StateWithUser>
 
 
 Class UserReplenishmentOrderAdapter has been removed and is no longer part of the public API.
-
+Use ReplenishmentOrderAdapter in @spartacus/order/core.
```
Comment (WeizhengSap) [inline docs/migration/5_0-generated-typescript-changes-doc.md]: `ReplenishmentOrderHistoryAdapter`

File: docs/migration/5_0-generated-typescript-changes-doc.md [hunk 18]
```diff
@@ -12640,7 +12610,7 @@ Class UserReplenishmentOrderAdapter has been removed and is no longer part of th
 
 
 Class UserReplenishmentOrderConnector has been removed and is no longer part of the public API.
-
+Use ReplenishmentOrderConnector in @spartacus/order/core.
```
Comment (WeizhengSap) [inline docs/migration/5_0-generated-typescript-changes-doc.md]: `ReplenishmentOrderHistoryConnector`

File: docs/migration/5_0-generated-typescript-changes-doc.md [hunk 19]
```diff
@@ -12649,7 +12619,7 @@ Class UserReplenishmentOrderConnector has been removed and is no longer part of
 
 
 Class UserReplenishmentOrderService has been removed and is no longer part of the public API.
-
+Use ReplenishmentOrderFacade in @spartacus/order/root instead.
```
Comment (WeizhengSap) [inline docs/migration/5_0-generated-typescript-changes-doc.md]: `ReplenishmentOrderHistoryFacade`

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts (hunk 1 above)
Comment (plabadie) [reply projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: Fixed.

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts (hunk 2 above)
Comment (plabadie) [reply projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: fixed.

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts (hunk 3 above)
Comment (plabadie) [reply projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: Fixed

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts (hunk 4 above)
Comment (plabadie) [reply projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: fixed

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts (hunk 5 above)
Comment (plabadie) [reply projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: fixed

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts (hunk 6 above)
Comment (plabadie) [reply projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: fixed

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts (hunk 10 above)
Comment (plabadie) [reply projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: fixed

File: projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts (hunk 9 above)
Comment (plabadie) [reply projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts]: fixed

**Review by WeizhengSap — APPROVED** [review]

**Pull request merged**